pontoon/base/models/entity.py

```diff
@@ -858,7 +858,20 @@ def for_project_locale(
             y = r"\y" if search_match_whole_word else ""
```

Comment:
```suggestion
            y = r"\y" if search_match_whole_word else ""
            r = "" if search_identifiers else "=.*"
            o = "" if search_identifiers else ".*"
```

Reply:
I don't have a better idea on what variable names to use, I went with p**r**efix and p**o**stfix. Which is lame.